lib/SILOptimizer/UtilityPasses/BugReducerTester.cpp

```diff
@@ -58,6 +58,12 @@ static llvm::cl::opt<FailureKind> TargetFailureKind(
         clEnumValEnd),
     llvm::cl::init(FailureKind::None));
 
+
+LLVM_ATTRIBUTE_NOINLINE
+void THIS_TEST_IS_EXPECTED_TO_CRASH_HERE() {
+  llvm_unreachable("Found the target!");
```

Comment:
`assert(false && "...");` or `abort();` would be more appropriate if you want to ensure there's a real crash here and not just a fall-off-the-edge.